src/migrations/tasks/20230327102038_twu-terms.ts

```diff
@@ -0,0 +1,38 @@
+import { generateUuid } from "back-end/lib";
+import { makeDomainLogger } from "back-end/lib/logger";
+import { console as consoleAdapter } from "back-end/lib/logger/adapters";
+import Knex from "knex";
+
+const logger = makeDomainLogger(consoleAdapter, "migrations");
+
+const slug = "team-with-us-terms-and-conditions";
```

Comment:
this is a duplicate of https://github.com/bcgov/digital_marketplace/pull/377/files#diff-f3b44cb6ba03d1053018cf98a71793369c924049cf8c44d06596098f63140e66R10 

the order of db migrations will require the sequence of merging to be `twu-proposals` followed by `twu-qualifications-tab`. Recommend removing content creation for 'team-with-us-terms-and-conditions' here.